initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -88,24 +135,30 @@ elif [ -e /etc/redhat-release -o -e /etc/SuSE-release ]; then
 fi
 
 echo "Removing Directory - ${path}"
-rm -rf ${path}
+removedir ${path}
+
+if [ -n "${mklink}" -a -n "${lpath}" ] ; then
```

Comment:
I don't think so.  I do not think there is any logic to determine if it was pre-existing and do something different.  I think its just rm or rm -r on the files listed in the package.  But I do not know for sure whats deep inside the rpm mechanism.